### PR TITLE
feat: add selected state for about steps

### PR DESCRIPTION
**Pull request by vm**
Screen Shot 2022-12-08 at 1 19 00 PM
Screen Shot 2022-12-08 at 1 18 56 PM

will add updated images in a future pr

### CODE REVIEW

File: src/pages/About/index.tsx [hunk 1]
```diff
@@ -149,10 +149,47 @@ const GetStarted = styled.div`
   margin-bottom: 100px;
 `
 
+const CARDS = [
+  {
+    to: '/swap',
```
Comment (zzmp) [inline src/pages/About/index.tsx:154]: Should these be hashed routes (eg `#/swap`)?

File: src/pages/About/Step.tsx [hunk 2]
```diff
@@ -31,18 +32,22 @@ const Step = ({
...
   index: number
   title: string
   description: string
   isLast?: boolean
+  onClick: () => void
+  expanded?: boolean
 }) => {
   return (
-    <StyledStep isLast={isLast}>
+    <StyledStep isLast={isLast} onClick={onClick}>
```
Comment (zzmp) [inline src/pages/About/Step.tsx:46]: Can `isLast` be moved to a css selector?

**Review by zzmp — APPROVED** [review]

File: src/pages/About/Step.tsx (hunk 2 above)
Comment (vm) [reply src/pages/About/Step.tsx:46]: good point yes

File: src/pages/About/index.tsx (hunk 1 above)
Comment (vm) [reply src/pages/About/index.tsx:154]: gonna fix all the routing in another pr

**Pull request merged**